next/components/meta.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { NextSeo } from "next-seo";
+import * as React from "react";
+
+type Metatag = {
+  tag: string;
+  attributes: {
+    name?: string;
+    content?: string;
+    href?: string;
+    rel?: string;
+  };
+};
+
+type Metatags = Metatag[];
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  metatags?: Metatags;
+}
+
+export function Meta({ title, metatags }: MetaProps) {
+  const router = useRouter();
+
+  const { t } = useTranslation();
+
+  const getTag = React.useCallback(
+    (str: string, key: keyof Metatag["attributes"] = "name") => {
+      const result = metatags?.find((tag) => tag.attributes[key] === str);
+      return result?.attributes;
+    },
+    [metatags]
+  );
+
+  // We want to determine if we need to add the language path
+  // to create the canonical link for this page:
+  const languagePathFragment =
+    router.locale === router.defaultLocale ? "" : `/${router.locale}`;
+
+  const data = {
+    title: getTag("title")?.content ?? title,
+    description: getTag("description")?.content ?? t("meta-site-description"),
+    canonical: `${process.env.NEXT_PUBLIC_FRONTEND_URL}${languagePathFragment}${
+      router.asPath !== "/" ? router.asPath : ""
+    }`,
+    imageSrc: getTag("image_src", "rel")?.href,
```

Comment:
do we take this out, or try to see how to provide that image?

Reply:
I'm not really sure about this. I'd lean towards leaving it out for now and adding it later if there's a need?

(I did search our GitHub https://github.com/search?l=JavaScript&q=org%3Awunderio+img_src&type=Code to see if there was a clear and tested way of doing it in Wunder, but didn't find anything.)

Reply:
I have added the necessary configuration in 1008b790ab754a6c19d6bc26548abddfdbafab9b for drupal to print the url of the field_image image file into the metatag. This works for articles only, since we don't have that field in other content types.

Next step would be maybe to provide a default image in the next frontend (public folder?) and print a link to that one in case the metatags from drupal don't include an image

Reply:
I added the fallback in f11837e67a19c0b8f6723ab59b8a9df90342e395